hokusai/staging.yml

```diff
@@ -56,22 +60,22 @@ metadata:
   name: force-web
   namespace: default
   annotations:
-    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: "arn:aws:iam::585031190124:server-certificate/2018-01-17_artsy-net-wildcard"
-    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
-    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
-    service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
-    service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: "true"
-    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "300"
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: 'arn:aws:iam::585031190124:server-certificate/2018-01-17_artsy-net-wildcard'
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: 'http'
+    service.beta.kubernetes.io/aws-load-balancer-ssl-ports: '443'
+    service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
+    service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: 'true'
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '300'
 spec:
   ports:
-  - port: 443
-    protocol: TCP
-    name: https
-    targetPort: 5000
-  - port: 80
-    protocol: TCP
-    name: http
-    targetPort: 5000
+    - port: 443
+      protocol: TCP
+      name: https
+      targetPort: 5000
+    - port: 80
+      protocol: TCP
+      name: http
+      targetPort: 5000
   selector:
     app: force
     layer: application
```

Comment:
prettier now fixes yml files, so now they're consistent 👍 

looks like it just doesn't like the implicit array after a key feature, seems reasonable to me.